src/schema/metadata/Genetics.yaml

```diff
@@ -0,0 +1,12 @@
+---
+key_name: Genetics
+description: |
+  An object containing information about the genetics descriptor.
+type: object
+properties:
+  Database:
+    $ref: Genetics.Database.yaml
+  Dataset:
+    $ref: Genetics.Dataset.yaml
+  Descriptors:
+    $ref: Genetics.Descriptors.yaml
```

Comment:
What does everyone think of this instead of defining those fields in here directly?

Reply:
looks clean to me